PublicSDKAPITester/RevenueCatAPI.m

```diff
@@ -0,0 +1,120 @@
+//
+//  RevenueCatAPI.m
+//  MigrateTester
+//
+//  Created by Joshua Liebowitz on 6/18/21.
+//
+
+#import "RevenueCatAPI.h"
+#import "Purchases.h"
+#import "RCPurchases.h"
+
+@import StoreKit;
+
+@implementation RevenueCatAPI
+
+bool canI;
+NSString *version;
+
+BOOL automaticAppleSearchAdsAttributionCollection;
+BOOL debugLogsEnabled;
+RCLogLevel logLevel;
+NSURL *proxyURL;
+BOOL forceUniversalAppStore;
+BOOL simulatesAskToBuyInSandbox;
+RCPurchases *sharedPurchases;
+BOOL isConfigured;
+BOOL allowSharingAppStoreAccount;
+BOOL finishTransactions;
+id<RCPurchasesDelegate> delegate;
+NSString *appUserID;
+BOOL isAnonymous;
+
++ (void)allTheThings {
```

Comment:
Just call all the things

Reply:
would this perhaps work better as a test target instead of an app?

Reply:
Maybe, https://github.com/RevenueCat/purchases-ios/issues/593